### PR TITLE
Added table download feature for bed file format

### PR DESCRIPTION
**Pull request by UAMS16**
image

### CODE REVIEW

**Comment by tncowart** [comment]
Can you include a screenshot and description?

**Review by tncowart — CHANGES_REQUESTED** [review]

File: cegs_portal/static/css/project.css [hunk 1]
```diff
@@ -1613,6 +1613,11 @@ input[type="submit"], input[type="button"], button {
   line-height: 2rem;
 }
 
+.text-lg {
+  font-size: 1.125rem;
+  line-height: 1.75rem;
+}
+
```
Comment (tncowart) [inline cegs_portal/static/css/project.css:1620]: Doesn't this need to be added to project.css.tw?

Comment (UAMS16) [reply cegs_portal/static/css/project.css:1620]: I regenerated the files when running the local server but no changes were detected in project.css.tw

File: cegs_portal/search/tsv_templates/v1/non_targeting_reos.py [hunk 2]
```diff
@@ -1,17 +1,48 @@
...
+        ]
+    )
+
+    for reo in reos:
+        for source in reo.sources.all():
+            row = [
+                source.chrom_name,
+                source.location.lower,
+                source.location.upper,
+                f"{source.chrom_name}:{source.location.lower}-{source.location.upper}:{if_strand(feature.strand)}:{feature.name}",
```
[the remark this answers is not in the document]
Comment (UAMS16) [reply cegs_portal/search/tsv_templates/v1/non_targeting_reos.py]: Completed

File: cegs_portal/search/templatetags/custom_helpers.py [hunk 3]
```diff
@@ -14,3 +14,11 @@ def format_float(value):
         return f"{value:.2e}"
     else:
         return f"{value:.6f}"
+
+
+@register.filter
```
[the remark this answers is not in the document]
Comment (UAMS16) [reply cegs_portal/search/templatetags/custom_helpers.py]: Completed

File: cegs_portal/search/views/custom_views.py [hunk 4]
```diff
@@ -155,9 +155,9 @@ def get(self, request, _options, data, *args, **kwargs):
     def get_json(self, _request, options, data, *args, **kwargs):
         return JsonResponse(self.__class__.json_renderer(data, options), safe=False)
 
-    def get_tsv(self, _request, _options, data, *args, **kwargs):
+    def get_tsv(self, _request, options, data, *args, **kwargs):
         filename = kwargs.get("filename", None)
-        return TsvResponse(self.__class__.tsv_renderer(data), filename)
+        return TsvResponse(self.__class__.tsv_renderer(data, options), filename)
```
[the remark this answers is not in the document]
Comment (UAMS16) [reply cegs_portal/search/views/custom_views.py:160]: Completed

**Review by tncowart — APPROVED** [review]
Looks good!

**Pull request merged**